lib/ansible/modules/packaging/os/yum.py

```diff
@@ -639,6 +639,42 @@ def local_envra(path):
                                header[rpm.RPMTAG_ARCH])
 
 
+def set_env_proxy(function):
+    def fetching(spec, module, conf_file, installroot):
+        # setting system proxy environment and saving old, if exists
+        yumb = yum_base(conf_file, installroot)
```

Comment:
I'm not sure why but other parts of the code are using "my" for the variable name for a yum_base object so we should keep doing that.

Reply:
fix